Upgrade-Firmware.sls

```diff
@@ -0,0 +1,26 @@
+Deploy_firmware_file:
+  file.managed:
+    - source: salt://FirmwareUpgrade/9056/cRIO-9056_8.0.0.cfg
+    - name: /sl_safemode/sl_safemode.cfg
```

Comment:
i would not call it sl_safemode so that we don't create a false ideea that this safemode is SL specific. just call it safemode. Also, can we deploy it under /tmp so that it gets deleted automatically after reboot? Is the tmpfs partition too small?

Reply:
That makes sense, I will rename it. I've started deploying it to the tmpfs but it could fill up quickly. This is why I am cleaning the salt cache as well, salt caches the file too. So I've moved away from this idea and relied on the bigger sized rootfs.

Reply:
And more than that, I will put "replace: True" to make sure we always replace the file with the desired one.